solr/solr-ref-guide/src/major-changes-in-solr-9.adoc

```diff
@@ -82,6 +82,9 @@ All the usages are replaced by BaseHttpSolrClient.RemoteSolrException and BaseHt
 
 * SOLR-11623: Every request handler in Solr now implements PermissionNameProvider. Any custom or 3rd party request handler must also do this
 
+* SOLR-15809: Get rid of blacklist/whitelist terminology. JWTAuthPlugin parameter `algWhitelist` is now `algAllowlist`. The old parameter will still
```

Comment:
maybe state that they will be removed in Solr 10? Or I guess we can wait for the Solr 10 upgrade notes.

Reply:
That can go in solr 10 when we remove support...